### PR TITLE
Documentation update for Android

### PR DESCRIPTION
**Pull request by timkimadobe**
## Description

Update documentation from iOS Edge Bridge to reflect Android syntax and links.

## Related Issue

See [MOB-16680](https://jira.corp.adobe.com/browse/MOB-16680) for additional info.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: Documentation/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+> It is recommended to send well formatted XDM data directly from an application using the [Edge.sendEvent](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension/edge-network-api-reference#sendevent) API instead of using the `trackState` and `trackAction` APIs with the Edge Bridge extension. However, in cases where it is not feasible to refactor the application, the Edge Bridge extension is available as a drop-in solution to send `trackState` and `trackAction` data to the Edge Network.
+>
+>  For new implementations of the SDK, it it highly recommended to send XDM data directly using the [`Edge.sendEvent`](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension/edge-network-api-reference#sendevent) API.
+>
+
+## Before starting
+
+### Install AEP Edge extension
+
+The Adobe Experience Platform Edge Bridge extension requires the Adobe Experience Platform Edge Network extension in order to operate. As a first step install and configure the [AEP Edge](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension) extension, then continue with the steps below.
```
Comment (timkimadobe) [inline Documentation/getting-started.md:14]: There was no aepsdk-edge-android repo so I linked to the gitbook docs for Android Edge instead

File: Documentation/api-reference.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Adobe Experience Platform Edge Bridge for Android
+
+## API Reference
+
+| APIs                                           		|
+| ---------------------------------------------- |
+| [extensionVersion](#extensionversion)	|
+| [registerExtension](#registerextension)	|
```
Comment (emdobrin) [inline Documentation/api-reference.md:8]: nit: table alignment

File: Documentation/api-reference.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+------
+
+### extensionVersion
+Returns the version of the Edge Bridge extension.
+
+#### Java
```
Comment (emdobrin) [inline Documentation/api-reference.md:15]: Any chance to add Kotlin examples as well (here and for registration)?

File: Documentation/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      MobileCore.setApplication(this);
+      try {
+        Edge.registerExtension();
+        EdgeBridge.registerExtension();
+        Identity.registerExtension();
+        // register other extensions
+        MobileCore.start(new AdobeCallback () {
+            @Override
+            public void call(Object o) {
+                MobileCore.configureWithAppID("yourAppId");
```
Comment (emdobrin) [inline Documentation/getting-started.md:62]: we can use "yourEnvironmentFileID"

File: Documentation/tutorials/map-track-data-using-data-prep.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+## Select data
+While creating a new datastream, select **Save and Add Mapping** to go to the **Select data** step directly.  Alternately, if using an already saved datastream, select **Edit Mapping**. From here, provide a sample JSON object that represents the data layer used in the application as sent from the Edge Bridge mobile extension.
+
+The following are examples of the JSON objects sent from the Edge Bridge mobile extension. The Experience event sent from the Edge Bridge extension contains both **xdm** and **data** JSON objects. The **data** object contains the context data passed to the `MobileCore.trackAction()` and `MobileCore.trackState()` APIs.
+
+The following is an example of an `MobileCore.trackAction` API call and the resulting JSON Experience event sent to the Edge Network. Note the JSON is truncated to show just the relevant data.
+```swift
+MobileCore.track(action: "add_to_cart", data: ["product.id": "12345", "product.add.event": "1", "product.name": "wide_brim_sunhat", "product.units": "1"])
+```
```
Comment (emdobrin) [inline Documentation/tutorials/map-track-data-using-data-prep.md:33]: This code-snippet needs to be converted to java as well (you can use the test app examples)

File: Documentation/tutorials/map-track-data-using-data-prep.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+}
+```
+
+</p></details>
+
+
+The following is an example of a `MobileCore.trackState` API call and the resulting JSON Experience event sent to the Edge Network. Note the JSON is truncated to show just the relevant data.
+```swift
+MobileCore.track(state: "hats/sunhat/wide_brim_sunhat_id12345", data: ["product.name": "wide_brim_sunhat", "product.id": "12345", "product.view.event": "1"])
+```
```
Comment (emdobrin) [inline Documentation/tutorials/map-track-data-using-data-prep.md:82]: Should be java as well

File: Documentation/tutorials/map-track-data-using-data-prep.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+The mapping process may be repeated for additional source data if needed by selecting the **Add JSON** button.
+
+![select data](../assets/select-data-trackaction.png)
+
+> **Note**
+> XDM source fields are automatically mapped if the same field appears in the target schema. For example, the fields _xdm.\_id_ and _xdm.timestamp_ are required fields in a time-series XDM schema so you will notice they are automatically mapped from the source data to the target schema and do not require a mapping entry.
+
+> **Note**
+> The Edge Bridge extension automatically sets an _xdm.eventType_ value of _analytics.track_. However, the value may be changed by adding a new mapping row in Data Prep by setting the **Target Field** to "eventType".
+> ![map eventType](../assets/map-eventtype.png)
```
Comment (emdobrin) [inline Documentation/tutorials/map-track-data-using-data-prep.md:164]: there seem to be a broken link here

File: Documentation/api-reference.md (hunk 2 above)
Comment (timkimadobe) [reply Documentation/api-reference.md]: Table alignment updated!

File: Documentation/api-reference.md (hunk 3 above)
Comment (timkimadobe) [reply Documentation/api-reference.md:15]: Added Kotlin examples to API reference for both methods

File: Documentation/getting-started.md (hunk 4 above)
Comment (timkimadobe) [reply Documentation/getting-started.md]: Updated to suggested string val!

File: Documentation/tutorials/map-track-data-using-data-prep.md (hunk 5 above)
Comment (timkimadobe) [reply Documentation/tutorials/map-track-data-using-data-prep.md]: Updated snippet for java and kotlin examples

File: Documentation/tutorials/map-track-data-using-data-prep.md (hunk 6 above)
Comment (timkimadobe) [reply Documentation/tutorials/map-track-data-using-data-prep.md]: Updated for java and kotlin examples

**Review by timkimadobe — COMMENTED** [review]
Thanks for the review @emdobrin! Updated documentation with java and kotlin syntax, and applied other suggestions

File: Documentation/tutorials/map-track-data-using-data-prep.md (hunk 7 above)
Comment (emdobrin) [reply Documentation/tutorials/map-track-data-using-data-prep.md]: This one still shows as broken, might need to take it outside of the note block to display correctly.

**Review by emdobrin — APPROVED** [review]
Looks great, left a small comment for the broken link.

**Pull request merged**